integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1611,6 +1617,158 @@ public static void createDomainAndVerify(Domain domain,
     return isVoyagerReady(namespace, podName);
   }
 
+  /**
+   * add security context constraints to the service account of db namespace.
+   * @param serviceAccount - service account to add to scc
+   * @param namespace - namespace to which the service account belongs
+   */
+  public static void addSccToDBSvcAccount(String serviceAccount, String namespace) {
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command("oc adm policy add-scc-to-user anyuid -z " + serviceAccount + " -n " + namespace))
+        .execute(), "oc expose service failed");
+  }
+
+  /**
+   * We need to expose the service as a route for ingress.
+   *
+   * @param serviceName - Name of the route and service
+   * @param namespace - Namespace where the route is exposed
+   */
+  public static String createRouteForOKD(String serviceName, String namespace) {
+    assertTrue(new Command()
+        .withParams(new CommandParams()
+            .command("oc expose service " + serviceName + " -n " + namespace))
+        .execute(), "oc expose service failed");
+    String command = "oc -n " + namespace + " get routes " + serviceName + "  '-o=jsonpath={.spec.host}'";
+
+    ExecResult result = Command.withParams(
+                         new CommandParams()
+                          .command(command))
+                          .executeAndReturnResult();
+
+    boolean success =
+            result != null
+            && result.exitValue() == 0
+            && result.stdout() != null
+            && result.stdout().contains(serviceName);
+
+    String outStr = "Did not get the route hostName \n";
+    outStr += ", command=\n{\n" + command + "\n}\n";
+    outStr += ", stderr=\n{\n" + (result != null ? result.stderr() : "") + "\n}\n";
+    outStr += ", stdout=\n{\n" + (result != null ? result.stdout() : "") + "\n}\n";
+
+    assertTrue(success, outStr);
+
+    getLogger().info("exitValue = {0}", result.exitValue());
+    getLogger().info("stdout = {0}", result.stdout());
+    getLogger().info("stderr = {0}", result.stderr());
+
+    String hostName = result.stdout();
+    getLogger().info("route hostname = {0}", hostName);
+
+    try {
```

Comment:
remove hard-coded sleep

Reply:
ok, There is no status for the route for me verify the route is operational. The route exists as soon as expose the service, but access to the app fails. So, I did look to see if there is a way to tell the status and have not found any. I see Thread.sleep() used even in our test classes and in this file even. So, I thought this could be one of those cases where we couldn't avoid using it.